### ISSUE TITLE
Clicking "Abort" and then "Back" overwrites the previous reply with the one stored in "Back"

### ISSUE THREAD

**Issue by NintendoManiac64** [LostRuins/lite.koboldai.net#41]
Title says it all really but, if that's still not enough, then here's a video demonstration (using the Linux Mint 21.2 Cinnamon live ISO):
[VIDEO GOES HERE]

**Comment by LostRuins** [comment]
Hmm yeah it is a bit odd, but that's due to the way the undo stack works currently. For now, you can just press "Back" twice and it should be ok.

**Comment by NintendoManiac64** [comment]
> you can just press "Back" twice and it should be ok.

No, very much no. Pressing back a second time just causes the previous reply before that to be removed.

**Comment by jojorne** [comment]
I believe that this was fixed together with https://github.com/LostRuins/koboldcpp/issues/531 since they are related issues.

**Comment by NintendoManiac64** [comment]
Just a heads up, the "Too many anonymous requests on using the AI Horde currently" error appearing causes the exact same issue to occur as if you had manually aborted the generation of the next message.

That's probably a bigger problem since that occurs on its own and can subsequently result in lost messages unless you start manually copying and pasting to combine the two back together.

**Comment by NintendoManiac64** [comment]
> I believe that this was fixed together with [LostRuins/koboldcpp#531](https://github.com/LostRuins/koboldcpp/issues/531) since they are related issues.

I'm quite late but this still occurs
